CHANGELOG.txt

```diff
@@ -47,6 +47,7 @@ v27.0.00
         Behaviour: updated the view so that followups are displayed as conversational logs and stored in a separate table
         Behaviour: added the ability to able to link a behaviour record to some other existing behaviour record
         Behaviour: added positive behaviour records to the Daily Behaviour Summary CLI script
+        Behaviour-Add-a-new-permission-so-that-teachers-can-only-view-behaviours-created-by-them
```

Comment:
This has come through a bit odd, maybe a copy-paste thing?

Reply:
Updated!! Thanks...